app/views/pages/tutorials/pages/pages.liquid

```diff
@@ -7,25 +7,39 @@ slug: tutorials/pages/pages
 searchable: true
 ---
 
-**Pages** are the most essential components of our platform, that define content displayed at a given path. All pages have to be located in the `views/pages` directory. Each page is represented by a single file with `liquid` extension.
+**Pages** are the most essential components of our platform, that define content displayed at a given path. All pages have to be located in the `views/pages` directory. Each page is represented by a single file with `liquid` extension. 
+
+Each page is accessible on the URL indicated by its location in the codebase if not otherwise specified in the page configuration `slug` property. For example, the page `app/views/pages/get-started/hello-world.liquid` deployed to the domain `http://example.com` will be automatically accessible at `http://example.com/get-started/hello-world`. 
```

Comment:
```suggestion
Each page is accessible on the URL indicated by its location in the codebase if not otherwise specified in the page configuration `slug` property. For example, the page `app/views/pages/get-started/hello-world.liquid` deployed to the domain `https://example.com` will be automatically accessible at `https://example.com/get-started/hello-world`. 
```